package.json

```diff
@@ -85,11 +84,12 @@
     "pdfkit": "^0.12.1",
     "prop-types": "^15.6.0",
     "qs": "^6.3.0",
-    "react": "^16.9.0",
```

Comment:
What is the reason for saying any React 16? (here are some features, such as hooks and <> that are not available at 16.0)

Reply:
I tried upgrading to version 17, but enzyme does not yet officially support React 17 (https://github.com/enzymejs/enzyme/issues/2429). However, we should upgrade to latest 16! Will do that now.

Reply:
Done in 93d62e8cd2b63f2cfed5f4501b3ecf9a2f387029